CHANGELOG.md

```diff
@@ -2,9 +2,13 @@
 
 ## v1.4.0 (2021-08-XX)
 

```

Comment:
We should start to document the maximum CE version usable to create RPKs for this PyPRT version:
I suggest to add "* Compatible CityEngine versions to create RPKs with: 2021.0 or earlier"